tests/qos/test_qos_sai.py

```diff
@@ -755,7 +757,7 @@ def testQosSaiSharedReservationSize(
         dst_asic_index = get_src_dst_asic_and_duts['dst_asic_index']
         src_testPortIps = dutConfig["testPortIps"][src_dut_index][src_asic_index]
         dst_testPortIps = dutConfig["testPortIps"][dst_dut_index][dst_asic_index]
-        (src_port_ids, dst_portids) = check_port_count
+        (src_port_ids, dst_portids) = check_skip_shared_res_test
```

Comment:
unpack fail, because of check_skip_shared_res_test() has three return value

try to fix in PR https://github.com/sonic-net/sonic-mgmt/pull/8644